.circleci/config.yml

```diff
@@ -43,10 +61,68 @@ jobs:
           key: uikit-yarn-dependencies-{{ checksum "yarn.lock" }}
           paths:
             - ./.yarn/cache
+  create-ticket:
+    macos:
+      xcode: "14.1"
+    steps:
+      - checkout
+      - clone_deployment_script
+      - run:
+          name: Extract release version
+          command: |
+            release_version=$(python3 ~/sdk-deployment/scripts/v1.2/extract_version.py --branch_name $CIRCLE_BRANCH)
+            echo "export RELEASE_VERSION=$release_version" >> $BASH_ENV
+            source $BASH_ENV
+      - run:
+          name: Jira - Create release version
+          command: |
+            python3 ~/sdk-deployment/scripts/v1.2/create_version.py\
+              --jira_auth_user $JIRA_AUTH_USER\
+              --jira_auth_api_token $JIRA_AUTH_API_TOKEN\
+              --project SDKRLSD\
+              --name ios_core@$RELEASE_VERSION\
+              --description 'Created by automation'
```

Comment:
Created by automation @ repo_name

Reply:
Handled in this commit https://github.com/sendbird/sendbird-uikit-react/pull/658/commits/0913283ced05d7ab0763586bb678271b812908f2 too